### ISSUE TITLE
respondsToSelector: typo

### ISSUE THREAD

**Issue by mobitar** [Instagram/IGInterfaceDataTable#14]
On line 140, you have 

```
else if ([dataSource respondsToSelector:@selector(table:configureHeaderController:)]) {
        [dataSource table:self configureFooterController:controller];
      }
```

when it should be 

```
else if ([dataSource respondsToSelector:@selector(table:configureFooterController:)]) {
        [dataSource table:self configureFooterController:controller];
      }
```

(changed `Header` to `Footer`)

**Comment by acoomans** [comment]
Fixed in https://github.com/Instagram/IGInterfaceDataTable/commit/e99e96469582ca50391f2cdfea64ada284038547

**Comment by fitcontrol** [comment]
Hi I'trying to develop fitness App, need help.
Do you know if coospo sdk is compatible with Instagram?
Thanks
El 2/5/2015 18:10, Arnaud Coomans <notifications@github.com> escribió:Closed #14.
